tools/mhddos/start.py

```diff
@@ -0,0 +1,1194 @@
+#!/usr/bin/env python3
+import argparse
+import random
+from contextlib import suppress
+from json import load
+from logging import basicConfig, getLogger, shutdown
+from math import log2, trunc
+from multiprocessing import RawValue
+from os import urandom as randbytes
+from pathlib import Path
+from random import choice as randchoice
+from re import compile
+from socket import (
+    AF_INET,
+    IPPROTO_TCP,
+    SOCK_DGRAM,
+    SOCK_RAW,
+    SOCK_STREAM,
+    TCP_NODELAY,
+    gethostname,
+    socket,
+)
+from ssl import CERT_NONE, SSLContext, create_default_context
+from sys import argv
+from sys import exit as _exit
+from threading import Event, Thread
+from time import sleep, time
+from typing import Any, List, Set
+from urllib import parse
+
+import urllib3
+import yarl
+from certifi import where
+from cloudscraper import create_scraper
+from dns import resolver
+from icmplib import ping
+from psutil import cpu_percent, net_io_counters, process_iter, virtual_memory
+from PyRoxy import Tools as ProxyTools
+from requests import Response, Session, cookies, get
```

Comment:
Here's the corrected version of your text:  

It's better to define imports in the following order:  
1. Imports from the standard library (e.g., `import random`)  
2. Imports from standard library submodules (e.g., `from datetime import datetime`)  
3. Imports of third-party modules (e.g., `import yarn`)  
4. Imports from third-party submodules (e.g., `from requests import Response`)  
5. Imports from the current application (e.g., `from . import ...`)

Reply:
We used black and isort for that... but as I see they don't do that. Maybe you know tools for this?

Reply:
isort should do that, maybe it has missed that file for some reason ? Currently it looks good